### PR TITLE
[IN-2191] Cleanup gems before install to prevent dependency issues (fixes fastlane install)

### PR DESCRIPTION
**Pull request by MrBlaise**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [ ] I updated the corresponding tests if there were any.
- [ ] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [x] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [x] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

### CODE REVIEW

**Comment by MrBlaise** [comment]
I tested it against a faulty linkfrom using ansible

File: roles/ruby-gems/tasks/main.yml [hunk 1]
```diff
@@ -1,4 +1,10 @@
 ---
+- name: Remove all (non default) gems
+  script: ../../../_scripts/ruby_cleanup.sh
```
Comment (bitrise-tom) [inline roles/ruby-gems/tasks/main.yml:3]: I would put the script under the files/ directory for this role

**Review by bitrise-tom — CHANGES_REQUESTED** [review]

**Review by lmesz-bitrise — COMMENTED** [review]
I see you tested on a faulty linkfrom but did you tested it on a generated one, or it will be tested during weekly generation?

**Comment by MrBlaise** [comment]
> I see you tested on a faulty linkfrom but did you tested it on a generated one, or it will be tested during weekly generation?

I tested this role on a faulty and it fixed the issue on it, I don't want to run it manually by hand on a clean linkfrom, it will be tested during the image generation once this merged.

File: roles/ruby-gems/tasks/main.yml (hunk 1 above)
Comment (MrBlaise) [reply roles/ruby-gems/tasks/main.yml]: Done

**Comment by lmesz-bitrise** [comment]
> > I see you tested on a faulty linkfrom but did you tested it on a generated one, or it will be tested during weekly generation?
> 
> I tested this role on a faulty and it fixed the issue on it, I don't want to run it manually by hand on a clean linkfrom, it will be tested during the image generation once this merged.

No not a clean linkfrom you already tested it by hand :) During generation, but alright will be tested after merge

**Review by lmesz-bitrise — APPROVED** [review]

**Review by lmesz-bitrise — APPROVED** [review]

**Review by lmesz-bitrise — APPROVED** [review]

**Review by bitrise-tom — APPROVED** [review]

**Review by peterheja-bitrise — APPROVED** [review]
LGTM

**Pull request merged**